include/secp256k1_frost.h

```diff
@@ -1,17 +1,110 @@
 #ifndef SECP256K1_FROST_H
 #define SECP256K1_FROST_H
 
+#include "secp256k1_extrakeys.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#include <stdint.h>
```

Comment:
It doesn't seem like this header is actually being used here? I think you meant to include stddef but that's already transitively included from `secp256k1.h`